### ISSUE TITLE
Separate out world creation

### ISSUE THREAD

**Issue by de-sh** [vyuham/rtrcrs#16]
Currently, the rendering system uses world generation logic embedded into the main program, this wouldn't allow for the use of portable models and ends up having unique values of colour rendered on each render. This is non-deterministic and undesirable to a certain extent, so I propose we explore separating out the logic for world generation, if possible.

CC: @bkp31415 @firebreath1001

**Comment by bkp31415** [comment]
Yes , i agree with you that we need a seperate world creation logic to use portable models and we can look into it to make it possible. :)
